docs/userguides/compile.md

```diff
@@ -85,7 +85,14 @@ compile:
 ## Settings
 
 Generally, configure compiler plugins using your `ape-config.yaml` file.
-For example, when using the `vyper` plugin, you can configure settings under the `vyper` key:
+One setting that applies to many compiler plugins is `compiler_cache_folder`, which holds dependency source files the compiler uses when compiling your contracts.
+By default, the folder is in your `contracts/.cache` folder but there are times you may want to move this to another location.
+
+```yaml
+compiler_cache_folder: .cache
```

Comment:
idea: we could move this to the `ape_compile` plugin, so then youd do this:

```yaml
compile:
    cache_folder: .cache
```

Reply:
I like the way this looks.  Just curious, is it generally kosher for plugins to reference other plugins' config?

Reply:
sometimes it makes sense, like in this case.

Reply:
other examples: many test providers, like ape-hardhat and ape-foundry, read stuff from the ape-test config

Reply:
Got a little weird, but got something working.  Check it out: [ff6a456](https://github.com/ApeWorX/ape/pull/1897/commits/ff6a4562432d6b17545f583342ea68665c4468f1)